ui/src/components/EntryDetailed/EntryViewer.tsx

```diff
@@ -21,7 +21,7 @@ const SectionsRepresentation: React.FC<any> = ({data, color, updateQuery}) => {
                     break;
                 case SectionTypes.SectionBody:
                     sections.push(
-                        <EntryBodySection key={i} color={color} content={row.data} updateQuery={updateQuery} encoding={row.encoding} contentType={row.mimeType} selector={row.selector}/>
+                        <EntryBodySection key={i} title={row.title} color={color} content={row.data} updateQuery={updateQuery} encoding={row.encoding} contentType={row.mimeType} selector={row.selector}/>
```

Comment:
same as previous message about this change as part of https://github.com/up9inc/mizu/pull/644

Reply:
@gadotroee doesn't matter. It required the same thing. We'll resolve the conflicts if any.